.gitignore

```diff
@@ -3,10 +3,7 @@
 #################################################
 
 .DS_Store
-.env
```

Comment:
I would keep the file ignored. This file is often used to store secrets and no longer ignoring it could mean that these could accidentally be committed and pushed to a public repository.

If you still want to include an initial copy of it with the checkout you could force commit it once.

Reply:
Fixed in the latest commit.